src/components/StakingVaults/StakingTable.tsx

```diff
@@ -36,7 +36,7 @@ export const StakingTable = ({ data, onClick, showTeaser }: StakingTableProps) =
         Cell: ({ row }: { row: RowProps }) => (
           <Skeleton isLoaded={row.original.isLoaded}>
             <AssetCell
-              assetId={row.original.assetId}
+              assetId={row.original.underlyingAssetId ?? row.original.assetId}
```

Comment:
Now that we have the notion of `underlyingAssetId` which properly reflects the underlying AssetId vs. the OpportunityId (either the LpId or StakingId i.e the farming / LP contract AccountId), this is the way to go